pocs/mount/__init__.py

```diff
@@ -1 +1,55 @@
+from glob import glob
+
 from pocs.mount.mount import AbstractMount  # pragma: no flakes
+from pocs.utils import error
+from pocs.utils import load_module
+from pocs.utils.location import create_location_from_config
+from pocs.utils.logger import get_root_logger
+
+
+def create_mount_from_config(config, mount_info=None, earth_location=None):
+    """ Sets up the mount that will be used by the observatory """
+
+    logger = get_root_logger()
+
+    if 'mount' not in config:
+        logger.info("No mount in config")
+        return None
+
+    if earth_location is None:
+        site_details = create_location_from_config(config)
+        earth_location = site_details['earth_location']
+
+    if mount_info is None:
+        mount_info = config.get('mount')
+
+    model = mount_info.get('model')
+
+    model = mount_info.get('brand')
```

Comment:
Sorry for the confusion, we have some things that are not named properly in the config.  Let's try to clear this up.  The `driver` is used to load the module so that is okay. Let's remove `brand` from here as well as from the [pocs.yaml](https://github.com/panoptes/POCS/blob/develop/conf_files/pocs.yaml#L43) config file.  But let's also change the `model` in the config to be `model: iOptron iEQ30 Pro`, which is just a descriptive name.

```suggestion
```